library/std/src/thread/scoped.rs

```diff
@@ -299,7 +301,7 @@ impl<'scope, T> ScopedJoinHandle<'scope, T> {
     }
 }
 
-impl<'env> fmt::Debug for Scope<'env> {
+impl<'scope, 'env> fmt::Debug for Scope<'scope, 'env> {
```

Comment:
Tiny debatable style nit, since `'scope` and `'env` don't play any role in this impl:

```suggestion
impl fmt::Debug for Scope<'_, '_> {
```